src/config/contracts.ts

```diff
@@ -1,17 +1,26 @@
+import {
+  NEXT_PUBLIC_ACCOUNT_MAINNET_ADDRESS,
+  NEXT_PUBLIC_ACCOUNT_TESTNET_ADDRESS,
+  NEXT_PUBLIC_MANAGER_MAINNET_ADDRESS,
+  NEXT_PUBLIC_MANAGER_TESTNET_ADDRESS,
+  NEXT_PUBLIC_STAKED_CELO_MAINNET_ADDRESS,
+  NEXT_PUBLIC_STAKED_CELO_TESTNET_ADDRESS,
+} from 'src/config/consts';
```

Comment:
Could we use relative paths ? From my experience absolute paths cause some issues in a long run. 

This can be configured in VS code - https://stackoverflow.com/questions/52432191/auto-import-in-visual-studio-code-only-offering-absolute-path-with-lerna-subpack

Reply:
We could, but i'd be relunctant to do so without changing the path for every other file to make it consistant. I feel that would be out of the scope of this PR.

Also, considereing all the nested directories, using relative path could make it more difficult when refactoring.

Reply:
I agree that if we would want to address it would make sense to do it in separate PR :-)